common/cuchemcommon/data/helper/chembldata.py

```diff
@@ -41,15 +41,16 @@
 
 
 def download_chembl_db():
-    setup_script = '/opt/nvidia/cheminfomatics/launch.sh'
-    if os.path.exists(setup_script):
-        logger.info('Triggering db setup...')
-        result = run(['bash', '-c',
-                        f'cd /opt/nvidia/cheminfomatics && {setup_script} setup'])
-        logger.info(f'Model download result: {result.stdout}')
-        logger.info(f'Model download result: {result.stderr}')
-        if result.returncode != 0:
-            raise Exception('Error downloading model')
+    pass
+    # setup_script = '/opt/nvidia/cheminfomatics/launch.sh'
```

Comment:
Was this code added where needed?
If so remove comment

Reply:
Need for this data download code is removed in clean up branch.